config/clouddriver.yml

```diff
@@ -101,7 +101,9 @@ dockerRegistry:
     - name: ${providers.dockerRegistry.primaryCredentials.name}
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
-      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile}
+      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
+      repositories:
+        - ${providers.dockerRegistry.primaryCredentials.repository}
```

Comment:
@lwander Hope you had a good holidays! What if I leave these two lines, but modify 'spinnaker.yml' and 'default-spinnaker-local.yml' from this:

> repository: ${SPINNAKER_DOCKER_REPOSITORY:library/nginx}

to this:

> repository: ${SPINNAKER_DOCKER_REPOSITORY:}

That way '/v2/_catalog' is still the default behavior, but the repository won't be ignored if it's specified.

Would that still break your public image?